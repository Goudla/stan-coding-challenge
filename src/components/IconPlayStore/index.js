import * as React from 'react';

// Assets
import playStoreSVG from './assets/play-store.svg';

class IconPlayStore extends React.PureComponent<*> {
  render() {
    return (
      <img src={playStoreSVG} alt="Play Store" width="135" />
    );
  }
}

export default IconPlayStore;
