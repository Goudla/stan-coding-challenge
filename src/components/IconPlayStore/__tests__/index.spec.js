import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

// Scenes
import IconPlayStore from '../';

describe('<IconPlayStore />', () => {
  it('Should render', () => {
    const renderer = new ShallowRenderer();
    renderer.render(<IconPlayStore />);
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
