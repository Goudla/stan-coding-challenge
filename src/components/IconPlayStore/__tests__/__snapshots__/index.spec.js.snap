// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<IconPlayStore /> Should render 1`] = `
<img
  alt="Play Store"
  src="play-store.svg"
  width="135"
/>
`;
